kong/runloop/balancer.lua

```diff
@@ -1036,6 +1048,43 @@ local function get_upstream_health(upstream_id)
 end
 
 
+--------------------------------------------------------------------------------
+-- Get healthcheck information for a balancer.
+-- @param upstream_id the id of the upstream.
+-- @return table with balancer health info
+local function get_balancer_health(upstream_id)
+
+  local balancer_health = {
```

Comment:
since this local is not used, you can just return it at the very end at once, eg:

```lua
return {
  health = ...,
  id = upstream_id,
}
```

Reply:
Done!